crates/proto/proto/app.proto

```diff
@@ -23,6 +23,39 @@ message OpenServerSettings {
     uint64 project_id = 1;
 }
 
+// Request sent from the remote host to ask the local Zed app
+// to open paths/urls in the current window (or a new one).
+message RemoteCliOpen {
```

Comment:
I'm pretty confused about this. My thought was that the remote cli would send a message over the socket to the remote server, which would handle actually opening the files/parsing the paths/etc. Then it'd communicate back to the client that a new buffer had been opened. The way this code currently works it seems to me like the server sends this open request back to the client, and then the client itself goes to look for that path locally (which will surely fail because the path was valid on the remote). Am I missing something here?